src/main/java/uk/gov/pay/ledger/transactionsummary/dao/TransactionSummaryDao.java

```diff
@@ -59,6 +63,17 @@
             "GROUP BY t.gateway_account_id, year, month " +
             "ORDER BY t.gateway_account_id, year, month";
 
+    private static final String PERFORMANCE_REPORT = "SELECT COALESCE(SUM(t.no_of_transactions),0) AS volume, " +
+            "COALESCE(SUM(t.total_amount_in_pence), 0) AS total_amount, " +
+            "COALESCE(SUM(t.total_amount_in_pence)/SUM(t.no_of_transactions), 0) AS avg_amount " +
+            "FROM transaction_summary t " +
+            "WHERE t.type= 'PAYMENT' " +
+            "AND t.live= TRUE";
+
+    private static final String WITH_STATE = " and t.state=:state";
```

Comment:
Shouldn't it be all uppercase AND to be consistent with the SQL?

Reply:
Done.